app/javascript/packages/document-capture/components/in-person-call-to-action.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import sinon from 'sinon';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { computeAccessibleDescription } from 'dom-accessibility-api';
+import { AnalyticsContextProvider } from '../context/analytics';
+import InPersonCallToAction from './in-person-call-to-action';
+
+describe('InPersonCallToAction', () => {
+  it('renders a section with label and description', () => {
+    const { getByRole } = render(<InPersonCallToAction />);
+
+    const section = getByRole('region', { name: 'in_person_proofing.headings.cta' });
+    const description = computeAccessibleDescription(section);
+
+    expect(description).to.equal('in_person_proofing.body.cta.new_feature');
```

Comment:
A couple other reasons for this:

- Consistency with how we assert in Ruby specs
- Ensuring that this would pass regardless of the current locale
- Because the locale data isn't actually loaded into the JavaScript text context (the actual rendered output of the components are the locale strings)